sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_users_services_first_seen_v1/query.sql

```diff
@@ -1,136 +1,99 @@
-WITH base AS (
+WITH fxa_events AS (
   SELECT
-    * REPLACE (
-      IF(service IS NULL AND event_type = 'fxa_activity - cert_signed', 'sync', service) AS service
-    )
+    timestamp,
+    user_id,
+    IF(service IS NULL AND event_type = 'fxa_activity - cert_signed', 'sync', service) AS service,
+    device_id,
+    os_name,
+    flow_id,
+    event_type,
+    country,
+    LANGUAGE,
+    entrypoint,
+    utm_term,
+    utm_medium,
+    utm_source,
+    utm_campaign,
+    utm_content,
+    ua_version,
+    ua_browser,
   FROM
-    firefox_accounts.fxa_content_auth_oauth_events
+    `moz-fx-data-shared-prod.firefox_accounts.fxa_content_auth_oauth_events`  -- # TODO: there seems to be fxa_content_auth_events, maybe we should use that instead since oauth has been deprecated for a long time now?
```

Comment:
@kik-kik I think you are the best person to make this call on which table to use.

Reply:
For now let's use the same view, but I think we should see how many "similar" views and tables we have and reduce them down to one to avoid having to maintain multiple things that are "basically" the same.

Reply:
assuming that oauth is gone, i'd recommend the `fxa_content_auth_stdout_events` view here

Reply:
Thanks @relud, another view to add to a list to review. I've noticed we have a few different views that seem to be doing the same thing. I will create a separate issue for investigating that and deprecating duplicate logic.